docs/src/tutorials/tutorial_4.md

```diff
@@ -0,0 +1,329 @@
+]),
+```
+
+with:
+
+```julia
+eye_centers = [Point(-40,-30), Point(40,-30)]
+nose = [O, Point(-10,20), Point(10, 20), O]
+```
+
+and
+
+```julia
+function eyes(centers, radius, color)
+    sethue(color)
+    circle.(centers, radius, :fill)
+    setcolor("white")
+    circle.(centers, radius/5, :fill)
+end
+```
+
+![Up to the nose](./assets/jarvis_nose.gif)
+
+## Using Transformations
+
+Let's give him some moving lips so he can communicate with us:
+
+```julia
+upper_lip = [Point(-40, 45), Point(40, 45)]
+lower_lip = [Point(-40, 55), Point(40, 55)]
+```
+
+These are just the outer points of the lips:
+
+```julia
+function lip(p1, p2)
+    setline(2)
+    move(p1)
+    c1 = p1 + Point(10, 10)
+    c2 = p2 + Point(-10, 10)
+    curve(c1, c2, p2)
+    do_action(:stroke)
+end
+```
+
+This function uses some more functions of the awesome Luxor package.
+
+The lips should be a little thicker than the other lines that we have drawn so far so let's set `setline(2)`. (default is 1).
+First we move to the starting point of the lip and create two control points a bit below and to the vertical center.
+
+The `curve` function is used to draw a cubic [Bézier curve](https://en.wikipedia.org/wiki/B%C3%A9zier_curve). 
+It unfortunately doesn't support the `:stroke` at the end so we have to do this with `do_action(:stroke)` this time.
+
+Now our two actions:
+
+```julia
+Action(60:150, (args...)->lip(upper_lip...); subactions=[
+    SubAction(1:15, appear(:fade)),
+    [SubAction(20i:20i+10, Translation(0, -5)) for i in 1:5]...,
+    [SubAction(20i+10:20i+20, Translation(0, 5)) for i in 1:5]...
+]),
+Action(60:150, (args...)->lip(lower_lip...); subactions=[
+    SubAction(1:15, appear(:fade)),
+    [SubAction(20i:20i+10, Translation(0, 5)) for i in 1:5]...,
+    [SubAction(20i+10:20i+20, Translation(0, -5)) for i in 1:5]...
+]),
+```
+
+Yeah I like those `...` splatting :smile:
+
+We fade them in at the beginning and then they shall move up and down a couple of times.
+
+Finally let him speak:
+
+```julia
+function speak(str)
+    fontsize(15)
+    text(str, Point(100, 50))
+end
+```
+
+
+```julia
+Action(80:120, (args...)->speak("I'm Jarvis"); subactions=[
+    SubAction(1:5, appear(:fade)),
+    SubAction(36:40, disappear(:fade)),
+]),
+Action(120:150, (args...)->speak("How are you?"); subactions=[
+    SubAction(1:5, appear(:fade)),
+    SubAction(36:40, disappear(:fade)),
+])
+```
+
+This time we also use the [`disappear`](@ref) function to fade out the text.
+
+
+![Jarvis](./assets/jarvis.gif)
+
+## Conclusion
+
+To recap, by working through this animation you should now:
+
+1. Understand how to make objects appear and disappear using subactions
+2. Be able to move objects inside subactions to have a finer control of the movement
+3. Know how to use splatting (the `...`) to create actions in a loop
+
+Hope you had as much fun reading this tutorial as I had creating our mascot.
+
+You're now ready to create your own big project.
```

Comment:
Just fun, maybe put a clown emoji here at the end? :clown_face:

Reply:
I have some font issues with some of those and wasn't able to find a one which works :smile: